messages/en.json

```
@@ -118,6 +118,32 @@
+    "CREATE_ITEM_SURVEY_FOR": "Create Survey for {itemName}",
+    "CREATE_GLOBAL_SURVEY": "Create Global Survey",
+    "CREATE_ITEM_SURVEY_DESC": "Create a survey specifically for this item",
+    "CREATE_GLOBAL_SURVEY_DESC": "Create a survey for all users",
+    "SURVEY_CREATED_SUCCESSFULLY": "Survey created successfully",
+    "FAILED_TO_CREATE_SURVEY": "Failed to create survey",
+    "SURVEY_FOR_ITEM": "Survey for {itemName}",
+    "SURVEYS_FOR_ITEM": "Surveys for {itemName}",
+    "COMPLETE_SURVEY_DESC": "Please complete this survey to share your feedback",
+    "MULTIPLE_SURVEYS_DESC": "Multiple surveys are available for this item",
+    "TAKE_SURVEY": "Take Survey",
+    "VIEW_DETAILS": "View Details",
```

Biome (2.1.2)

[error] The key VIEW_DETAILS was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)